movieland_domain/src/main/java/com/solovey/movieland/entity/reporting/ReportTopUser.java

```diff
@@ -0,0 +1,40 @@
+package com.solovey.movieland.entity.reporting;
+
+public class ReportTopUser {
```

Comment:
I think this entity will contain single top user so would be better to name it TopUser